### PR TITLE
Update deps to make it work on Node 18

### LINKED ISSUE #20: Compatibility with Node v16+
Hey there,

I am running the following code in a docker container with Canvas successfully installed (at least my understanding based on install steps mentioned in other issues).

`import { pdfToPng } from 'pdf-to-png-converter';`
`import { readFileSync } from 'node:fs';`

`a = readFileSync('routes/test_pdf/1650457668823.pdf')`
`pngPages = pdfToPng( a, {viewportScale: 1.0})`

I am getting the following error and can't for the life of me understand whats wrong.

`node: ../../nan/nan_object_wrap.h:32: static T* Nan::ObjectWrap::Unwrap(v8::Local<v8::Object>) [with T = Context2d]: Assertion object->InternalFieldCount() > 0' failed. Aborted`

Any help would be greatly appreciated.

### PR DESCRIPTION
**Pull request by EHadoux**
This PR fixes #20.
node-canvas has been released with pre-built binaries for Node 18, which was the last bit left to make it runnable on all platforms.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,19 +40,19 @@
     "tsc": "tsc --pretty"
   },
   "dependencies": {
-    "canvas": "2.10.1",
-    "pdfjs-dist": "2.16.105"
+    "canvas": ">=2.10.2",
```
Comment (sternam) [inline package.json:43]:
```suggestion
    "canvas": "2.10.2",
```

File: package.json [hunk 2]
```diff
@@ -40,19 +40,19 @@
...
+    "pdfjs-dist": "^3.0.0"
```
Comment (sternam) [inline package.json:44]:
```suggestion
    "pdfjs-dist": "3.0.0"
```

**Review by sternam — CHANGES_REQUESTED** [review]
@dichovsky @EHadoux This looks good to me, just one minor comment is my suggested changes: it is preferable to pin non-dev dependencies to specific versions.

**Comment by EHadoux** [comment]
Hey, all done, although I don't think it's preferable at all. Not pinning canvas would have removed the need for this PR. Now we have `overrides` in package.json so it's fine, but we don't benefit from the tests of this repo.

**Review by sternam — APPROVED** [review]
@dichovsky Could you please merge or give write ability to me or @EHadoux ?  This is holding up my migration to Node v18

> Hey, all done, although I don't think it's preferable at all. Not pinning canvas would have removed the need for this PR. Now we have `overrides` in package.json so it's fine, but we don't benefit from the tests of this repo.

💯 Thanks @EHadoux!  I think it is best practice what you're doing now: to pin to specific version for `dependencies` but use the `^` in devDependencies.

**Pull request merged**